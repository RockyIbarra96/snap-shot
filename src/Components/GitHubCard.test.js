import 'punycode';
import React from 'react';
import renderer from 'react-test-renderer';
import GHCard from './GitHubCard';



test('renders a snapshot', () => {
  const tree = renderer.create(<GHCard />).toJSON();
  expect(tree).toMatchSnapshot();
});

