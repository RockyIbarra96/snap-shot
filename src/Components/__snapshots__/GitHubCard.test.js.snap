// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      RockyIbarra96
    </div>
    <p
      className="card-text"
    >
      Hi, I'm Rocky and I like to party and code.
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Go to GitHub Profile
    </button>
  </div>
</div>
`;
